src/Kdyby/Translation/DI/TranslationExtension.php

```diff
@@ -264,15 +264,14 @@ public function beforeCompile()
 		$this->beforeCompileLogging($config);
 
 		$registerToLatte = function (Nette\DI\ServiceDefinition $def) {
-			$def
-				->addSetup('?->onCompile[] = function($engine) { Kdyby\Translation\Latte\TranslateMacros::install($engine->getCompiler()); }', ['@self'])
-				->addSetup('addFilter', ['translate', [$this->prefix('@helpers'), 'translate']]);
+			$def->addSetup('?->onCompile[] = function($engine) { Kdyby\Translation\Latte\TranslateMacros::install($engine->getCompiler()); }', ['@self']);
 
 			if (method_exists('Latte\Engine', 'addProvider')) { // Nette 2.4
-				$def->addSetup('addProvider', ['translator', '@Kdyby\Translation\Translator']);
-
+				$def->addSetup('addProvider', ['translator', [$this->prefix('@helpers'), 'getTranslator']])
```

Comment:
I'm [not sure](https://github.com/Kdyby/Translation/commit/7ae78c74e85b8a2966d807d30f5017f6f2a8aea7#diff-0614d9c5b2a16c577483bacde0b13568R160) :))